### DISCUSSION TITLE
Recognize

### DISCUSSION THREAD

**Discussion by bennypowers** [nextcloud/all-in-one#4340] [Ideas] [answered]
The Memories app uses Recognize to run facial recognition. Installing Recognize for AIO is not straightforward. It would be very nice to have a preconfigured container for various architectures.

**Comment by szaimen** [comment, marked as answer]
Hi, this is tracked here https://github.com/nextcloud/recognize/issues/1061 and here https://github.com/nextcloud/recognize/issues/73
